src/config.ps1

```diff
@@ -0,0 +1,95 @@
+function ConvertTo-HashTable {
+	param (
+		[Parameter(ValueFromPipeline)]
+		[PSCustomObject]$Object
+	)
+	$Table = @{}
+	$Object.PSObject.Properties | ForEach-Object {
+		$V = $_.Value
+		if ($V -is [Array]) {
+			$V = [System.Collections.ArrayList]$V
+		} elseif ($V -is [PSCustomObject]) {
+			$V = ($V | ConvertTo-HashTable)
+		}
+		$Table.($_.Name) = $V
+	}
+	return $Table
+}
+
+function GetPwrHome {
+	if ($PwrHome) {
+		$PwrHome
+	} elseif ($env:PwrHome) {
+		$env:PwrHome
```

Comment:
Same vein as below, do we want this to be `$env:AirpowerHome`?

Reply:
What about `$env:AirpowerPath`, similar to `$env:GoPath`, since the command is not in this directory anymore, doesn't seem correct to call it the home.

Reply:
I like `$env:AirpowerPath`

Reply:
Done